src/components/modules/toolbar/inline.ts

```diff
@@ -96,12 +96,17 @@ export default class InlineToolbar extends Module {
     if (!this.toolsInstances || this.toolsInstances.size === 0) {
       const allTools = this.inlineTools;
 
-      this.toolsInstances = new Map();
-      for (const tool in allTools) {
-        if (allTools.hasOwnProperty(tool)) {
-          this.toolsInstances.set(tool, allTools[tool]);
-        }
-      }
+      const orderedTools = new Map();
+
+      // get the user order specifications
+      const order = this.config.tools.header.inlineToolbar;
```

Comment:
you should not hardcode Header tool. Instead of this, get current Block's tool. 

User can specify different `inlineToolbar` options for all tools, or not to specify it at all. So, it seems like we need to reorder the Inline Toolbar on every opening.

To optimize calculations, you can cache toolbar order for each tool.

Reply:
Okay